app/templates/account.hbs

```diff
@@ -2,25 +2,21 @@
 
   <header>
 
-  {{#if config.billingEndpoint}}
-    <div class="cta-btn">
-    {{#if subscribeButtonInfo.subscribed}}
-      <a class="btn btn-activated" href={{billingUrl}}>
-        Subscription active!
-      </a>
-    {{else}}
-      {{#if subscribeButtonInfo.education}}
-        <a class="btn btn-activated" href={{billingUrl}}>
-          Educational account!
-        </a>
+  {{#unless config.enterprise}}
+    {{#if config.billingEndpoint}}
```

Comment:
All of the places that already have checks for things like `billingEndpoint` could be fixed by not setting the defaults in `config/environment.js`: https://github.com/travis-ci/travis-web/blob/1d0fb6468e83f6b99d16f15e1bd2a095e67c6380/config/environment.js#L51

In order to not set these things we could do `if (process.env.TRAVIS_PRO && !process.env.TRAVIS_ENTERPRISE) {` [here](https://github.com/travis-ci/travis-web/blob/1d0fb6468e83f6b99d16f15e1bd2a095e67c6380/config/environment.js#L40).